### DISCUSSION TITLE
Write patched boot.img failed

### DISCUSSION THREAD

**Discussion by Mahantor** [bmax121/APatch#426] [Q&A] [answered]
When I try to patch my boot ROM file, I encounter the error Write patched boot.img failed
![Screenshot_2024-05-02-22-52-59-802_me bmax apatch](https://github.com/bmax121/APatch/assets/91434001/37d95cff-6ff6-4c1a-b7a1-20412ba9aaf1)

![Screenshot_2024-05-02-22-53-15-359-edit_me bmax apatch](https://github.com/bmax121/APatch/assets/91434001/5a48dbc5-d097-4006-9f94-8691bc126e5d)

[APatch_bugreport_2024-05-02_22_58.tar.gz](https://github.com/bmax121/APatch/files/15193315/APatch_bugreport_2024-05-02_22_58.tar.gz)

**Comment by marat2509** [comment, marked as answer]
Apparently your kernel doesn't have the necessary things that require full APatch to work. 

Read more at https://apatch.top/requirements.html

**Reply by ballparkme** [reply]
This page no longer exists, could you provide a new one?

**Reply by marat2509** [reply]
https://apatch.dev/install.html#install-requirements

**Reply by Obnomus** [reply]
> Apparently your kernel doesn't have the necessary things that require full APatch to work.
> 
> Read more at https://apatch.top/requirements.html

I typed the command and it showed CONFIG_KALLSYMS=y as output but still it shows failed to patch boot.Img and on my other device it works fine

**Reply by ckravig** [reply]
> > Apparently your kernel doesn't have the necessary things that require full APatch to work.
> > Read more at https://apatch.top/requirements.html
> 
> I typed the command and it showed CONFIG_KALLSYMS=y as output but still it shows failed to patch boot.Img and on my other device it works fine

I ran the same commands and also returned all "y"s.

**Comment by Darlin19902015** [comment]
Ayuda tengo ya instalado una versión de Apatch pero al momento de actualizar a su última versión cuando voy a parchar me salta un error 
![Screenshot_2024-10-18-14-37-11-275_me bmax apatch](https://github.com/user-attachments/assets/4177e17b-d3db-4a6b-8948-064e168bb255)

**Reply by Obnomus** [reply]
Tbh I haven't updated to the latest version either so I can't say anything.
